kubeadm/flannel/README.md

```diff
@@ -0,0 +1,19 @@
+
+# Build your own image
+```
+build-arg:
+    <servercoreTag>: default ltsc2019
+    <cniVersion>:    default 0.8.5
+    <golangTag>:     default windowsservercore-1809
+```
+
+## Example
+### windows server 1809
+```bash
+docker build --build-arg servercoreTag=ltsc2019 --build-arg cniVersion=0.8.5 --build-arg golangTag=windowsservercore-1809 -t flannel:0.12.0-windowsservercore-1809 .
+```
+
+### windows server core 1909
+```bash
+docker build --build-arg servercoreTag=1909 --build-arg cniVersion=0.8.5 --build-arg golangTag=windowsservercore-1909 -t flannel:0.12.0-windowsservercore-1909 .

```

Comment:
There doesn't seem to be a golang windowsservercore-1909 image

Reply:
Subsequent glang will support windowsservercore-1909 image

Reply:
Until then won't this not work? I'm okay with it being just documentation on how to build your own image, but shouldn't we show how to build the Go 1909 image too?

Reply:
@benmoss Sure, No problem